### PR TITLE
Update deployment.rst

### PR DESCRIPTION
**Pull request by jepster**

### CODE REVIEW

File: deployment.rst
Comment (dominikhajduk) [inline deployment.rst]: @jepster is tag checkout more common or popular than pull master branch? I think it depend on methodology. I met both approach and I can't say that one of them is much more popular.

Comment (jepster) [inline deployment.rst]: @dominikhajduk Checking out the master branch is safer. Since you can rollback more clear to a previous release tag in case of failure. Git tags are the standard in module releases in the Drupal project.

Comment (dominikhajduk) [inline deployment.rst]: @jepster 
> Checking out the master branch is safer. Since you can rollback more clear to a previous release tag in case of failure.

Exactly. That's why I have doubts if checkout tag instead of checkout master and just pull new version from origin should be proposed here. Of course we both agree that tag should be created anyway.

**Comment by jepster** [comment]
Then let's propose the proper way. ;)

Dominik Hajduk <notifications@github.com> schrieb am Fr., 6. Jan. 2017
19:01:

**Comment by dominikhajduk** [comment]
@jepster Version from http://symfony.com/doc/current/deployment.html is fine for me. 
>If you're using source control (e.g. Git or SVN), you can simplify by having your live installation also be a copy of your repository. When you're ready to upgrade it is as simple as fetching the latest updates from your source control system.

Maybe just: "as **fetching** the latest updates" > "as **pulling** the latest updates" or "as **checkout** the latest updates".
But this is just my opinion and someone from core reviewers should decide.

**Review by javiereguiluz — APPROVED** [review]

**Comment by xabbuh** [comment]
Thank you @jepster.

**Pull request closed without merging**
